Sources/Instructions/Helpers/Public/Enums/CoachMarkNextLabelPosition.swift

```diff
@@ -0,0 +1,6 @@
+public enum CoachMarkNextLabelPosition {
+    case topRight
+    case topLeft
+    case bottomRight
+    case bottomLeft
```

Comment:
I'd suggest using the leading/trailing terminology, so that it's direction-agnostic. Let's add leading & trailing in it as well!

```suggestion
    case leading
    case trailing
    case topTrailing
    case topLeading
    case bottomTrailing
    case bottomLeading
```

Reply:
Hi @ephread! Thanks for the feedback!
The leading/trailing terminology seems good to me!
I have made some changes at 89584d2, feel free to check it out. :)